webhooks-extension/config/extension-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhooks-extension
+  labels:
+    app: webhooks-extension
+    tekton-dashboard-extension: "true"
+  annotations:
+    tekton-dashboard-display-name: webhooks-extension
+    tekton-dashboard-endpoints: "webhooks-extension/webhooks"
+spec:
+  type: LoadBalancer
```

Comment:
Has to be LoadBalancer? Why not NodePort as an example that'll work for everyone (e.g. those without a LoadBalancer) then if they want one they can change to LoadBalancer?